platform/linuxbsd/key_mapping_x11.cpp

```diff
@@ -109,16 +109,16 @@ static _XTranslatePair _xkeysym_to_keycode[] = {
 	{ XK_KP_8, Key::KP_8 },
 	{ XK_KP_9, Key::KP_9 },
 	// same but with numlock
-	{ XK_KP_Insert, Key::KP_0 },
-	{ XK_KP_End, Key::KP_1 },
-	{ XK_KP_Down, Key::KP_2 },
-	{ XK_KP_Page_Down, Key::KP_3 },
-	{ XK_KP_Left, Key::KP_4 },
-	{ XK_KP_Begin, Key::KP_5 },
-	{ XK_KP_Right, Key::KP_6 },
-	{ XK_KP_Home, Key::KP_7 },
-	{ XK_KP_Up, Key::KP_8 },
-	{ XK_KP_Page_Up, Key::KP_9 },
+	{ XK_KP_Insert, Key::INSERT },
+	{ XK_KP_End, Key::END },
+	{ XK_KP_Down, Key::DOWN },
+	{ XK_KP_Page_Down, Key::PAGEDOWN },
+	{ XK_KP_Left, Key::LEFT },
+	{ XK_KP_Begin, Key::KP_5 }, // not sure what this should be, doesn't do anything if left like this
```

Comment:
It's interesting that it doesn't do anything instead of inserting `5`.

X11 seems to define `XK_KP_Begin` as "BOL" which I suppose means "begin of line" as opposed to "EOL" which is "end of line". So I think they want it to be interpreted the same way as `Key::HOME` like `XK_KP_Begin`?

That being said, I'm on X11 / KDE and neither GTK+3 apps (Firefox) nor Qt5 ones (Dolphin, Konsole) seem to do anything with `XK_KP_Begin`. The only application I found where it seems to do _something_ is `xterm` where it triggers a weird refresh (the terminal flashes from light theme to dark theme for a second).

So I think the standard expectation should indeed be that this key doesn't do anything.
I'd suggest documenting it like this:
```suggestion
	// X11 documents this (numpad 5) as "begin of line" but no toolkit seems to interpret it this way.
	//{ XK_KP_Begin, Key::NONE },
```

Reply:
done but changed to Key::CLEAR as suggested below